arenadata/Dockerfile.ubuntu

```
@@ -29,27 +29,42 @@ RUN set -eux; \
     pip2 install allure-behave==2.4.0; \
     pip2 cache purge
 
+RUN set -eux; wget https://www.python.org/ftp/python/3.9.19/Python-3.9.19.tgz -O - | tar zxf -; \
+    cd Python-3.9.19; \
+    ./configure --prefix=/usr/local --enable-shared --enable-optimizations; \
+    make -j6; \
+    make install; \
+    echo "/usr/local/lib" >>/etc/ld.so.conf.d/gpdb.conf; \
+    ldconfig; \
+    cd ..; \
+    rm -rf Python-3.9.19.tgz Python-3.9.19
+
 WORKDIR /home/gpadmin
 
-FROM base as build
+FROM base AS build
 
 COPY . gpdb_src
 
 RUN mkdir /home/gpadmin/bin_gpdb
 
 ENV TARGET_OS=ubuntu
+ENV PYTHON=/usr/bin/python2
 ENV OUTPUT_ARTIFACT_DIR=bin_gpdb
 ENV CONFIGURE_FLAGS="--enable-debug-extensions --with-gssapi --enable-cassert --enable-debug --enable-depend" 
 
 # Compile with running mocking tests
+RUN bash /home/gpadmin/gpdb_src/concourse/scripts/compile_gpdb.bash && \
+    cd gpdb_src/src/pl/plpython && \
+    make clean
```

use new style and pushd and popd
```suggestion
RUN set -eux; \
    bash /home/gpadmin/gpdb_src/concourse/scripts/compile_gpdb.bash; \
    pushd gpdb_src/src/pl/plpython; \
    make clean; \
    popd
```